src/simulators.jl

```diff
@@ -29,40 +29,38 @@ Not currently compatible with automatic differentiation using Zygote.
 - `max_steps::Int=1000`: the maximum number of steps.
 - `tol::F=1000.0u"kJ * mol^-1 * nm^-1"`: the maximum force below which to
     finish minimization.
-- `run_loggers::Bool=false`: whether to run the loggers during minimization.
 - `log_stream::L=devnull`: stream to print minimization progress to.
 """
 struct SteepestDescentMinimizer{D, F, L}
     step_size::D
     max_steps::Int
     tol::F
-    run_loggers::Bool
     log_stream::L
 end
 
 function SteepestDescentMinimizer(;
                                     step_size=0.01u"nm",
                                     max_steps=1_000,
                                     tol=1000.0u"kJ * mol^-1 * nm^-1",
-                                    run_loggers=false,
                                     log_stream=devnull)
-    return SteepestDescentMinimizer(step_size, max_steps, tol, run_loggers, log_stream)
+    return SteepestDescentMinimizer(step_size, max_steps, tol, log_stream)
 end
 
 """
-    simulate!(system, simulator, n_steps; n_threads=Threads.nthreads())
-    simulate!(system, simulator; n_threads=Threads.nthreads())
+    simulate!(system, simulator, n_steps; n_threads=Threads.nthreads(), run_loggers = Bool)
```

Comment:
`run_loggers=true`. Then below could add "`run_loggers` is `true` by default except for `SteepestDescentMinimizer`".